samples/new-node.yaml

```diff
@@ -0,0 +1,17 @@
+# Use this manifest to announce the new node to the apiserver.
+# You need to fill the needed SSH connection information.
+# CMD: sudo kubectl --kubeconfig /etc/kubernetes/admin.conf create -n "kube-system" -f new-node.yaml
+apiVersion: v1
+kind: Node
+metadata:
+  name: <NODE_NAME>
+  annotations:
+    metalk8s.scality.com/ssh-user: <SSH_USER>
+    metalk8s.scality.com/ssh-port: <SSH_PORT>
+    metalk8s.scality.com/ssh-host: <IP_ADDRESS>
+    metalk8s.scality.com/ssh-key-path: <PATH/TO/SSH/KEY>
+    metalk8s.scality.com/ssh-sudo: <true if the user is not root> 
+spec:
+  taints:
+  - effect: NoSchedule
+    key: node-role.kubernetes.io/master
```

Comment:
We need labels as well, which is how the automation (Salt) can know the desired role of a node, not based on taints (which aren't necessarily required).

Reply:
any suggested labels for now ?